scipy/stats/tests/test_fit.py

```diff
@@ -394,6 +400,21 @@ def test_argus(self):
 
         assert_nllf_less_or_close(dist, data, res.params, shapes, **self.tols)
 
+    def test_beta(self):
```

Comment:
This is basically a copy-paste of the other distribution-specific tests; there's nothing special about it. I didn't have to tune the `shape` parameters or anything.